src/app/eslint/index.js

```diff
@@ -0,0 +1,61 @@
+import fs from 'fs'
+import path from 'path'
+import {
+  addScripts,
+  createPackageJson,
+  installDependencies
+} from 'src/utils/npm'
+import { actionText, dim } from 'src/utils/log-style'
+import { askLintChoices, getLintChoices } from 'src/utils/prompts'
+
+function copyFiles(eslintConfig) {
+  const templates = path.join(__dirname, '../src/app/eslint/templates')
+  const eslintPath = path.join(templates, 'eslint.js')
+  const prettierPath = path.join(templates, 'prettier.js')
+  const eslintContent = fs.readFileSync(eslintPath, 'utf8')
+
+  fs.copyFileSync(prettierPath, '.prettierrc.js')
```

Comment:
fs.promises is much nicer to work with if you're good using async calls

Reply:
You're right, I should improve write/read functions to work in parallel when possible.